src/Controller/Component/AuthComponent.php

```diff
@@ -349,7 +349,7 @@ protected function _unauthenticated(Controller $controller)
         }
 
         if (!empty($this->_config['ajaxLogin'])) {
-            $controller->viewPath = 'Element';
+            $controller->getview()->viewPath = 'Element';
```

Comment:
That's ok, but slowly moving away from direct access seems like a good idea to me